### DISCUSSION TITLE
How to adjust the interface scaling ratio or font size？

### DISCUSSION THREAD

**Discussion by CookieAteMe** [Eugeny/tabby#9502] [Q&A] [answered]
Hello everyone！
My OS is Ubuntu 22.04 jammy , the version of tabby is 1.0.2.7.
I encountered some problems while using it：
The font adjustment in the settings can only adjust the font in the terminal
and How to adjust the size or proportion of these fonts in the settings
![image](https://github.com/Eugeny/tabby/assets/38611633/7474b60b-b859-4ada-9ece-d33ee22ae587)
![image](https://github.com/Eugeny/tabby/assets/38611633/76267a57-b732-4920-bf19-0b2251c4fadf)

**Comment by moemoechu** [comment, marked as answer]
check plugin [background](https://github.com/moemoechu/tabby-background), there is a setting to change UI font size.
